google/cloud/bigquery/job.py

```diff
@@ -20,6 +20,9 @@
 import copy
 import re
 import threading
+import tqdm
```

Comment:
This needs to be an optional import. See:

https://github.com/googleapis/python-bigquery/blob/20f473bfff5ae98377f5d9cdf18bfe5554d86ff4/google/cloud/bigquery/table.py#L39-L42